### PR TITLE
Introduce enums for the severity

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

### CODE REVIEW

File: api/v1/model/src/commonMain/kotlin/Severity.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ */
+enum class Severity {
+    /** Used for issues that provide information only. */
+    HINT,
+
+    /** Used for issues that could lead to problems. */
+    WARNING,
+
+    /** Used for issues that must be addressed. */
+    ERROR;
```
Comment (sschuberth) [inline api/v1/model/src/commonMain/kotlin/Severity.kt]: Nit: The semicolon can be omitted if no functions follow.

File: model/src/commonMain/kotlin/Severity.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ */
+enum class Severity {
+    /** Used for issues that provide information only. */
+    HINT,
+
+    /** Used for issues that could lead to problems. */
+    WARNING,
+
+    /** Used for issues that must be addressed. */
+    ERROR;
```
Comment (sschuberth) [inline model/src/commonMain/kotlin/Severity.kt]: Nit: The semicolon can be omitted if no functions follow.

File: api/v1/model/src/commonMain/kotlin/Severity.kt [hunk 3]
```diff
@@ -30,5 +30,5 @@ enum class Severity {
     WARNING,
 
     /** Used for issues that must be addressed. */
-    ERROR;
+    ERROR
```
Comment (sschuberth) [inline api/v1/model/src/commonMain/kotlin/Severity.kt:33]: Wrong commit 😉

File: model/src/commonMain/kotlin/Severity.kt [hunk 4]
```diff
@@ -32,5 +32,5 @@ enum class Severity {
     WARNING,
 
     /** Used for issues that must be addressed. */
-    ERROR;
+    ERROR
```
Comment (sschuberth) [inline model/src/commonMain/kotlin/Severity.kt:35]: Wrong commit 😉

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: workers/reporter/src/test/kotlin/reporter/ReporterRunnerTest.kt
Comment (nnobelis) [inline workers/reporter/src/test/kotlin/reporter/ReporterRunnerTest.kt:1]: Does this change belong to this commit ?

Comment (mnonnenmacher) [reply workers/reporter/src/test/kotlin/reporter/ReporterRunnerTest.kt:1]: Moved it to the previous commit.

**Review by nnobelis — APPROVED** [review]

**Pull request merged**